lib/screens/explore_screens/vote_screens/delegate/delegates_tab/delegates_tab.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:seeds/components/full_page_error_indicator.dart';
+import 'package:seeds/components/full_page_loading_indicator.dart';
+import 'package:seeds/components/snack_bar_info.dart';
+import 'package:seeds/domain-shared/page_command.dart';
+import 'package:seeds/domain-shared/page_state.dart';
+import 'package:seeds/domain-shared/ui_constants.dart';
+import 'package:seeds/navigation/navigation_service.dart';
+import 'package:seeds/screens/explore_screens/vote_screens/delegate/delegates_tab/components/delegate_card.dart';
+import 'package:seeds/screens/explore_screens/vote_screens/delegate/delegates_tab/components/introducing_delegates_dialog.dart';
+import 'package:seeds/screens/explore_screens/vote_screens/delegate/delegates_tab/components/remove_delegate_confirmation_dialog.dart';
+import 'package:seeds/screens/explore_screens/vote_screens/delegate/delegates_tab/interactor/viewmodels/delegate_bloc.dart';
+import 'package:seeds/screens/explore_screens/vote_screens/delegate/delegates_tab/interactor/viewmodels/delegate_event.dart';
+import 'package:seeds/screens/explore_screens/vote_screens/delegate/delegates_tab/interactor/viewmodels/delegate_page_commands.dart';
+import 'package:seeds/screens/explore_screens/vote_screens/delegate/delegates_tab/interactor/viewmodels/delegate_state.dart';
+
+class DelegatesTab extends StatelessWidget {
+  const DelegatesTab({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
+      create: (context) => DelegateBloc()..add(const LoadDelegateData()),
+      child: BlocConsumer<DelegateBloc, DelegateState>(
+        listener: (context, state) {
+          final pageCommand = state.pageCommand;
+
+          if (pageCommand is ShowDelegateRemovalConfirmation) {
+            showDialog<void>(
+              context: context,
+              barrierDismissible: false,
+              builder: (_) {
+                return BlocProvider.value(
+                  value: BlocProvider.of<DelegateBloc>(context),
+                  child: const RemoveDelegateConfirmationDialog(),
+                );
+              },
+            );
+          } else if (pageCommand is ShowOnboardingDelegate) {
+            showDialog<void>(
+              context: context,
+              barrierDismissible: false,
+              builder: (_) => const IntroducingDelegatesDialog(),
+            );
+          } else if (pageCommand is ShowErrorMessage) {
+            SnackBarInfo(pageCommand.message, ScaffoldMessenger.of(context)).show();
+          }
+
+          BlocProvider.of<DelegateBloc>(context).add(ClearPageCommand());
+        },
+        builder: (context, DelegateState state) {
```

Comment:
This is redundant you don't need to declare the state type here, that is defined in the Consumer sign decalration  `BlocConsumer<DelegateBloc, DelegateState>(`

change   `builder: (context, DelegateState state) {` to   `builder: (context, state) {`

Reply:
Fixed